creme/initializers.py

```diff
@@ -0,0 +1,106 @@
+"""
+Initializers module
+"""
+from collections import defaultdict
+from functools import partial
+import numpy as np
+from sklearn import utils
+
+__all__ = [
+    "Zeros",

```

Comment:
Could you replace simple quote per double quote? :-)

Reply:
Good point !

Reply:
Nice :+1: